test/basic-test.html

```diff
@@ -273,6 +273,20 @@
         expect(keys.keyboardEventMatchesKeys(event, 'up')).to.be.equal(true);
       });
     });
+
+    suite('matching keyboard events to top row and number pad digit keys', function() {
+      test('top row can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 49;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
+      });
+
+      test('number pad digits can be done imperatively', function() {
+        var event = new CustomEvent('keydown');
+        event.keyCode = 97;
+        expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
```

Comment:
You could extract a helper function for some of this tests like:
```javascript
function expectKeyboardEventMatchesKeys(keyCode){
  var event = new CustomEvent('keydown');
  event.keyCode = keyCode;
  expect(keys.keyboardEventMatchesKeys(event, '1')).to.be.equal(true);
}
```

and rewrite your unit test to something like that
```javascript
test('number pad digits can be done imperatively', function() {
  expectKeyboardEventMatchesKeys(97);
});
```